WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/WatchDog.java

```diff
@@ -2,31 +2,39 @@
 
 import com.intellij.icons.AllIcons;
 import com.intellij.ide.actions.ViewToolWindowButtonsAction;
+import com.intellij.ide.plugins.PluginManager;
 import com.intellij.openapi.components.ProjectComponent;
+import com.intellij.openapi.extensions.PluginId;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.ui.DialogWrapper;
 import com.intellij.openapi.ui.Messages;
+import nl.tudelft.watchdog.core.logic.network.JsonTransferer;
+import nl.tudelft.watchdog.core.logic.network.ServerCommunicationException;
+import nl.tudelft.watchdog.core.ui.wizards.User;
 import nl.tudelft.watchdog.logic.InitializationManager;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
 import nl.tudelft.watchdog.ui.preferences.Preferences;
-import nl.tudelft.watchdog.ui.preferences.ProjectPreferenceSetting;
+import nl.tudelft.watchdog.core.ui.preferences.ProjectPreferenceSetting;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectRegistrationStep;
 import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectRegistrationWizard;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectSliderStep;
 import nl.tudelft.watchdog.ui.wizards.userregistration.UserProjectRegistrationWizard;
-import nl.tudelft.watchdog.util.WatchDogGlobals;
-import nl.tudelft.watchdog.util.WatchDogLogger;
+import nl.tudelft.watchdog.core.util.WatchDogGlobals;
+import nl.tudelft.watchdog.core.util.WatchDogLogger;
 import nl.tudelft.watchdog.util.WatchDogUtils;
 import org.jetbrains.annotations.NotNull;
 
 public class WatchDog implements ProjectComponent {
     public static Project project;
 
-    /** The warning displayed when WatchDog is not active. */
-    public static final String WATCHDOG_INACTIVE_WARNING = "<html>WatchDog only works when you register a (possibly anonymous) user and project.<br><br>Takes less than one minute,  and you can win prices. As a registered user, you decide where WatchDog is active.";
+    /**
+     * The warning displayed when WatchDog is not registered.
+     */
```

Comment:
Actually, I don't think it's a good idea - in IJ we use different kind of labels (where html tags are the only way to create a new line, while in Eclipse \n is enough), so I think this should stay like this for now.